Header header
int32 rotorPosition
